discipl/Issues/demoVIL/description.xml

```diff
@@ -0,0 +1,105 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Virtueel Inkomstenloket</en>
+		<nl>Demo Virtueel Inkomstenloket</nl>
+	</Name>
+	<ShortName>
+		<en>Demo VIL</en>
+		<nl>Demo VIL</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>discipl</IssuerID>
+	<CredentialID>demoVIL</CredentialID>
+	<Description>
+		<en>Demo Virtueel Inkomstenloket (VIL) using Rule Services</en>
+		<nl>Demo Virtueel Inkomstenloket (VIL) die gebruik maakt van Regel Services</nl>
+	</Description>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.discipl.demoVIL.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.discipl.demoVIL.html</nl>
+	</IssueURL>
+
+	<Attributes>
+		<Attribute id="woonplaats">
+			<Name>
+				<en>Residence</en>
+				<nl>Woonplaats</nl>
+			</Name>
+			<Description>
+				<en>Residence</en>
+				<nl>Woonplaats</nl>
+			</Description>
+		</Attribute>
+		<Attribute id="geboortedatum">
+			<Name>
+				<en>Date of birth</en>
+				<nl>Geboortedatum</nl>
+			</Name>
+			<Description>
+				<en>Date of birth</en>
+				<nl>Geboortedatum</nl>
+			</Description>
+		</Attribute>
+		</Attribute>
```

Comment:
There is a double closing tag here.

Additionally, would you please run `irma scheme sign` in the root folder of this repository? After that, this should be ready to merge.

Reply:
Removed the closing tag en tried running `irma scheme sign`...
```
steven@steven-Surface-Laptop-3 ~/Development/irma-demo-schememanager (add_discipl_attributes_for_VIL) $ irma scheme sign
irma: command not found
```

Reply:
Ah, then you don't have the [`irma` commandline tool](https://irma.app/docs/irma-cli/) installed. That's alright, I've done it myself 🙂